torch/utils/benchmark/utils/timer.py

```diff
@@ -269,17 +275,16 @@ def blocked_autorange(
     ) -> common.Measurement:
         """Measure many replicates while keeping timer overhead to a minimum.
 
-        At a high level, blocked_autorange executes the following pseudo-code:
-        ```
-        `setup`
-
-        total_time = 0
-        while total_time < min_run_time
-            start = timer()
-            for _ in range(block_size):
-                `stmt`
-            total_time += (timer() - start)
-        ```
+        At a high level, blocked_autorange executes the following pseudo-code:::
```

Comment:
nit: just `::` should be enough according to docs, will still render `:`. Does it not work?

Reply:
Ah, it does indeed.